server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -85,6 +86,15 @@ export function create(
             // Summary information
             const summary = request.body.summary;
 
+            // Session information
+            const ordererUrl = config.get("worker:serverUrl");
```

Comment:
`ordererUrl` and `historianUrl` are used in multiple routes but are not dependent on anything within an individual request. I suggest hoisting them out of the route definitions to around where `enforceServerGeneratedDocumentId` is defined on line 39.

Reply:
Got it. Done.